nvm.sh

```diff
@@ -1443,7 +1443,13 @@ nvm_get_os() {
   NVM_UNAME="$(command uname -a)"
   local NVM_OS
   case "$NVM_UNAME" in
-    Linux\ *) NVM_OS=linux ;;
+    Linux\ *)
+      if [ -f "/etc/alpine-release" ]; then
```

Comment:
this seems like a brittle test - couldn't there be an Alpine without this?

Reply:
What about using `/etc/issue`? Though I see no reason to touch them both.

FYR:
```
$ for ver in 2.6 2.7 3.1 3.2 3.3 3.4 3.5 3.6 3.7 3.8; do docker run --rm -it alpine:$ver grep 'Alpine Linux' /etc/issue; done
Welcome to Alpine Linux 2.6
Welcome to Alpine Linux 2.7
Welcome to Alpine Linux 3.1
Welcome to Alpine Linux 3.2
Welcome to Alpine Linux 3.3
Welcome to Alpine Linux 3.4
Welcome to Alpine Linux 3.5
Welcome to Alpine Linux 3.6
Welcome to Alpine Linux 3.7
Welcome to Alpine Linux 3.8
```